websrc/default_hands/src/default_hands_main.tsx

```diff
@@ -974,23 +1041,47 @@ class DefaultHands extends React.Component< {}, {} >
 						]
 					} volume={ emptyVolume() } ref={ this.gadgetRegistryRef } />
 				</AvOrigin>
+
 				<AvOrigin path="/user/hand/left">
-					<AvTransform translateZ={ 0.04 } translateY={ 0.02 }>
-						<AvGrabButton onClick={ this.toggleGadgetMenu } 
-							modelUri={ g_builtinModelGear }/>
+					{ this.displayIntro && 
+					<AvHeadFacingTransform>
+						<AvTransform uniformScale = {0.07}>
+							<AvModel uri = {g_builtinModelMenuIntro}/>
+						</AvTransform>
+					</AvHeadFacingTransform>
+					}
+
+
+					<AvTransform uniformScale = {0.03} transform = {volumeDictionary.get(this.controllerType).leftHandTop}>
```

Comment:
This is going to fail badly for unknown interaction profiles. I would wrap this up in a function that returns some kind of placeholder/default volume for unknown profiles.